package-lock.json

```diff
@@ -1,8010 +1,128 @@
 {
   "name": "amazon-chime-sdk-smart-video-sending-demo",
   "version": "1.0.0",
-  "lockfileVersion": 2,
```

Comment:
Due to the local file version, should we use NPM 7 and then update this?

Reply:
Done. thanks